qubes/storage/__init__.py

```diff
@@ -40,6 +40,8 @@
 
 STORAGE_ENTRY_POINT = 'qubes.storage'
 
+_am_root = os.getuid() == 0
+_big_buffer = b'\0' * 4096
```

Comment:
Can you make it bigger? For example ISO9660 has its header at 32KiB offset. I don't mean one wants to keep ISO9660 on volatile volume, but that 4KiB isn't universally enough to ensure no collision with some filesystem (detection tool).
Maybe 1MiB is indeed too excessive, but lets make it at least 64KiB.